code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -237,6 +237,15 @@
 	return ..()
 
 
+/mob/living/carbon/human/transferAllItemToLoc(atom/newloc)
+	var/obj/item/item
+	for(var/slot in SLOT_ALL)
+		item = get_item_by_slot(slot)
+		if (item)
+			doUnEquip(item)
+			item.removed_from_inventory()
```

Comment:
temporarilyRemoveItemFromInventory

Reply:
Like i said, no this proc is when you are really removing from all the slots, so to the ground or to another inventory.
No to be confused with dropped(), wich should be named temporarilyRemoveItemFromInventory